Sign_In_Page.py

```diff
@@ -0,0 +1,23 @@
+from selenium import webdriver
+from selenium.webdriver.chrome.options import Options
+import Secure 
+import config
+import time
+
+
+class Sign_In_Page():
+
+    def __init__(self,driver):
+        self.driver = driver
+        driver.get(config.signInPage_url)
+    
+
+    def click_sign_in_button(self):
+        for cookie in Secure.cookies_coinbase: self.driver.add_cookie(cookie)
```

Comment:
After you fix the secure.py issue mentioned before, this should read `for cookie in secure.cookies.coinbase: self.driver.add_cookie(cookie)`